### DISCUSSION TITLE
Annotation info

### DISCUSSION THREAD

**Discussion by ea4aoj** [jekhokie/raspberry-noaa-v2#470] [General]
Hello, I have a question, when my satellite images are generated, a logo appears at the top and below the annotation info. I have seen that a colleague who has downloaded the raspberry image comes out as a gray sticker with all the info, but I get the logo and below the info in text. I have followed all the steps in the github repository and installed it on a debian armhf machine, but I am not using the precompiled image. Another thing I see is that in the configuration file it does not seem like antenna info to me, and this colleague who has the raspberry image yes, I do not know why there are differences in this regard, I have done a git pull and everything continues equal. What intrigues me the most is the annotation info at the top, because they are different, I put an image of myself and another of the one my friend is using. Thanks
![NOAA-19-20211113-201842-MCIR](https://user-images.githubusercontent.com/29492233/141659760-2963d07b-3381-425c-950d-391efbc5afae.jpg)
![photo_2021-11-13_22-38-56](https://user-images.githubusercontent.com/29492233/141659766-6a98072b-8795-4083-98c5-cee8a74fbbd4.jpg)
.

**Comment by ea4aoj** [comment]
OK, ya me contestaron en DISCORD. Gracias.

**Comment by dom-robinson** [comment]
I think this might help: [https://github.com/jekhokie/raspberry-noaa-v2/blob/master/docs/annotation.md](https://github.com/jekhokie/raspberry-noaa-v2/blob/master/docs/annotation.md) ?
